### ISSUE TITLE
Add scroll bar

### ISSUE THREAD

**Issue by physkets** [krawieck/lemmur#175]
### Is your feature request related to a problem? Please describe.
The lack of a scroll bar means that users have no idea how much of the post remains to be scrolled.

### Describe the solution you'd like
A simple scroll bar that's either permanent or disappears when not scrolling.

**Comment by pioneer** [comment]
Makes sense but IMO only partially because of the infinite scroll. Anyway it's useful to know how much I have already scrolled from the top.

**Comment by physkets** [comment]
I don't mean for there to be a scroll bar in the "post-list view". It makes sense though, within a post, where the number of comments is finite.
